hrms/hr/doctype/interview/interview.py

```diff
@@ -217,8 +217,14 @@ def send_daily_feedback_reminder():
 	interview_feedback_template = frappe.get_doc(
 		"Email Template", reminder_settings.feedback_reminder_notification_template
 	)
+
 	interviews = frappe.get_all(
-		"Interview", filters={"status": ["in", ["Under Review", "Pending"]], "docstatus": ["!=", 2]}
+		"Interview", filters={
+			"status": "Under Review", 
+			"docstatus": ["!=", 2], 
+			"scheduled_on": ["<=", getdate()], 
+			"to_time": ["<=", now_time()],
+		}
```

Comment:
```suggestion
		"Interview", 
		filters={
			"status": "Under Review", 
			"docstatus": ["!=", 2], 
			"scheduled_on": ["<=", getdate()], 
			"to_time": ["<=", nowtime()],
		}
```
One more formatting issue and the function name is nowtime(). My bad. Can you give the reviewer permission to edit the branch?

Reply:
Okay